### ISSUE TITLE
plugin methods to rewind and forward

### ISSUE THREAD

**Issue by vista136** [pejonic/cordova-pejonic-plugin-exoplayer#8]
whats a method to rewind and forward, on method ?

**Comment by johnnwanosike1** [comment]
For now is not available as the plugin was intended for live streaming!

**Comment by frontyard** [comment]
@vista136 yes it's possible but you should really be using the original plugin at https://github.com/frontyard/cordova-plugin-exoplayer

This one is a copy.
